app/src/main/res/values/strings.xml

```diff
@@ -203,12 +203,11 @@
 
     <!-- About -->
     <string name="about_title">About</string>
-    <string name="about_summary">A backup application using Android\'s internal backup API.</string>
+    <string name="about_summary">A backup application using Android\'s internal backup API. It is Free Software, released under Apache 2 license.\n\nLike all software, Seedvault may contain bugs or vulnerabilities.</string>
     <string name="about_version">Version: %s</string>
-    <string name="about_license">License: <a href="https://www.apache.org/licenses/LICENSE-2.0">Apache2</a></string>
-    <string name="about_author">Written by: <a href="https://github.com/stevesoltys">Steve Soltys</a> and <a href="https://blog.grobox.de">Torsten Grote</a></string>
-    <string name="about_design">Design by: <a href="https://www.glennsorrentino.com/">Glenn Sorrentino</a></string>
-    <string name="about_sponsor">Sponsored by: <a href="https://www.calyxinstitute.org">Calyx Institute</a> for use in <a href="https://calyxos.org">CalyxOS</a></string>
-    <string name="about_source_code">Source code: https://github.com/seedvault-app/seedvault</string>
+    <string name="about_contributor_headline">Contributors</string>
+    <string name="about_contributor_content">Steve Soltys, Creator\nTorsten Grote, Maintainer\nGlenn Sorrentino, Product Designer\nChirayu Desai, Android Engineer\nMichael Bestas, Android Engineer\nAayush Gupta, Android Engineer\nTranslation team from Weblate</string>
```

Comment:
I like the way that this is structured, but these names might change as people who change contribute, making it hard to stay up to date. It'd make things harder for translations too - in fact this already looks hard to translate.

I'd suggest just having a link to https://github.com/seedvault-app/seedvault/graphs/contributors instead - as that will always reflect the true state of the contributors.

Reply:
did change it to this link